src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -102,10 +103,21 @@ export const useFormSend = () => {
 
         toast({
           title: translate('modals.send.sent', { asset: data.asset.name }),
-          description: translate('modals.send.youHaveSent', {
-            amount: data.cryptoAmount,
-            symbol: data.cryptoSymbol
-          }),
+          description: (
+            <Text>
+              <Text>
+                {translate('modals.send.youHaveSent', {
+                  amount: data.cryptoAmount,
+                  symbol: data.cryptoSymbol
+                })}
+              </Text>
+              {data.asset.explorerTxLink && (
+                <Link href={`${data.asset.explorerTxLink}${broadcastTXID}`} isExternal>
+                  View in Explorer <ExternalLinkIcon mx='2px' />
```

Comment:
@abaqai can you add translations here please? see the `translate(...` function used in a lot of other UI elements

Reply:
Yup, will do! Is there an automatic way to fill in the translations for the non english messages.json.

Reply:
Unfortunately we have no way to automatically add the missing translations, nor are they expected to be present in every PR for now AFAIK. They should be present in `messages-en.json` however